Install dir:
    /usr/$(get_libdir)/rrdtool/ifOctets.tcl
    /usr/$(get_libdir)/rrdtool/pkgIndex.tcl

Due to the following change, pkglib_{DATA,SCRIPTS} is invalid:
    http://git.savannah.gnu.org/cgit/automake.git/commit/?id=9ca632642b006ac6b0fc4ce0ae5b34023faa8cbf

---
 bindings/tcl/Makefile.am |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/bindings/tcl/Makefile.am b/bindings/tcl/Makefile.am
index c0e8b0f..b7205e7 100644
--- a/bindings/tcl/Makefile.am
+++ b/bindings/tcl/Makefile.am
@@ -27,8 +27,9 @@ tclpkgdir = @TCL_PACKAGE_DIR@
 tclpkg_DATA = pkgIndex.tcl
 tclpkg_SCRIPTS = ifOctets.tcl
 else
-pkglib_DATA = pkgIndex.tcl
-pkglib_SCRIPTS = ifOctets.tcl
+tclpkgdir = $(libdir)/@PACKAGE@
+tclpkg_DATA = pkgIndex.tcl
+tclpkg_SCRIPTS = ifOctets.tcl
 endif
 
 # Automake doen't like `tclrrd$(VERSION)$(TCL_SHLIB_SUFFIX)' as
